test/integration/cli/test/index.test.js

```diff
@@ -48,7 +48,17 @@ describe('CLI Usage', () => {
         new RegExp(`Next\\.js v${pkg.version.replace(/\./g, '\\.')}`)
       )
     })
+
+    test('invalid directory', async () => {
+      const output = await runNextCommand(['non-existent'], {
+        stderr: true,
+      })
+      expect(output.stderr).toContain(
+        'Invalid project directory provided, no such directory'
```

Comment:
In the case when no command is provided, we could also print a help screen.

For example `next buidl` could print:

```
Invalid project directory provided `buildl`, no such directory.

Did you mean to run a subcommand instead?

next build
next start
next export
```

Reply:
Hmm, maybe we can add detecting mis-spellings of the main commands here, will investigate in a follow-up

Reply:
Opened PR adding this here https://github.com/vercel/next.js/pull/34836